mne/viz/topomap.py

```diff
@@ -3063,7 +3063,7 @@ def plot_regression_weights(model, ch_type=None, vmin=None, vmax=None,
 
     nrows = model.coef_.shape[1]
     ncols = len(ch_types)
-    fig, axes = plt.subplots(nrows, ncols, squeeze=False, layout='tight',
```

Comment:
this is likely to affect plots from users no?

can we avoid this change just by adding the right subplots_adjust in the example?

Reply:
Could you clarify? I removed the `layout` parameter here because old matplotlib doesn't support it. Instead, I put an `subplits_adjust` in the example to fix the layout of the subplot. Isn't that what you are suggesting here?

Reply:
I think you two are saying the same thing: avoid calling tight layout in package code, and (if necessary) use subplots_adjust in tutorial code.

Reply:
Ok. One thing though: [I put a call to `subplots_adjust` in the `plot_regression_weights` function](https://github.com/mne-tools/mne-python/pull/11046/files#diff-505965f399e13f9a66c6671ec34ab14816c84c6ad22dbdb37e1a94c9475c7d97R3085). I did this because it bothered me that if you just call `EOGRegression.plot()`, the axes overlap. Are we ok with that?

Reply:
> I put a call to `subplots_adjust` in the `plot_regression_weights` function.

I just took a look and left a couple of review comments relevant to this point.

Reply:
tnx